master/internal/api_runs.go

```diff
@@ -403,3 +406,87 @@ func (a *apiServer) MoveRuns(
 	}
 	return &apiv1.MoveRunsResponse{Results: results}, nil
 }
+
+// TODO (corban): this should be included in GetRun.
```

Comment:
is this TODO happening?

Reply:
No, since we don't currently have a run details page, I'm going to drop this TODO.